doc/FAQ.md

```diff
@@ -75,6 +76,13 @@ On your Edit/Manage Profiles page, you will find a "change profile photo" link.
 Clicking this will take you to a page where you can upload a photograph and select which profile it will be associated with.
 To avoid privacy leakage, we only display the photograph associated with your default profile as the avatar in your posts.
 
+<a name="language"></a>
+### How can I view Friendica in a certain language?
+
+You can do this by adding the `lang` parameter to the url in your url bar. The data in the parameter is a [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) code. eg. https://social.example.com/profile/example translated in German becomes https://social.example.com/profile/example?lang=de. If the question mark is already in the url you need to do it using a ampersand. eg. https://social.example.com/profile/example?tab=profile becomes https://social.example.com/profile/example&lang=de.
```

Comment:
Could you and the reviewers please remember, that there is one rule for the docs, which is one line per sentence.

Reply:
Ah drat, forgot again, sorry.